library/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 option(USE_STATIC_MBEDTLS_LIBRARY "Build mbed TLS static library." ON)
 option(USE_SHARED_MBEDTLS_LIBRARY "Build mbed TLS shared library." OFF)
 option(LINK_WITH_PTHREAD "Explicitly link mbed TLS library to pthread." OFF)
+option(LINK_WITH_TRUSTED_STORAGE "Explicitly link mbed TLS library to trusted_storage." ON)
```

Comment:
This should default to OFF, as most of our Linux systems don't have a trusted_storage library available. On Mbed Linux, this can be changed to ON.

Reply:
done